src/logic/prepare-test.ts

```diff
@@ -0,0 +1,184 @@
+import {
+  ComlinkListLiteralNode,
+  ComlinkObjectLiteralNode,
+  ComlinkPrimitiveLiteralNode,
+  isUseCaseDefinitionNode,
+  UseCaseExampleNode,
+} from '@superfaceai/ast';
+import { SuperJson } from '@superfaceai/one-sdk';
+import { join as joinPath } from 'path';
+
+import { LogCallback } from '..';
+import { exists, mkdir } from '../common/io';
+import { OutputStream } from '../common/output-stream';
+import { ProfileId } from '../common/profile';
+import { testTemplate } from '../templates/testTemplate';
+import { loadProfile } from './publish.utils';
+
+export type ExampleInput = {
+  exampleKind: 'success' | 'error';
+  input: string;
+};
+
+/**
+ * Prepares string with number of \t according of passed parameter
+ * @param numberOfTabs number of \t in created string
+ * @returns
+ */
+function intend(numberOfTabs: number): string {
+  let res = '';
+  for (let i = 0; i < numberOfTabs; i++) {
+    res += '\t';
+  }
+
+  return res;
+}
```

Comment:
first, why tabs? :scream: 
second: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

Reply:
Yees, let the tabs flow through you 😈